faq/how.md

```diff
@@ -0,0 +1,13 @@
+---
+title: How does SCN Work?
+parent: FAQ
+nav_order: 4
+---
+
+# How Does the Seattle Community Network Work?
+
+The Seattle Community Network partners with the University of Washington to share free or low-cost internet access with areas of higher need.
+
+The Seattle Community Network functions by utilizing a portion of the University of Washington's internet bandwidth and sharing that access wirelessly with certain areas covered by the network. This is accomplished by using high gain wireless antennas to transmit data from the hardwired connection to a local node, which is then shared to nearby devices using the LTE (cell-phone data) standard. Individual users can then connect to this signal using an SCN-provided hardware, creating a local WiFi network to use for internet access in their home.
```

Comment:
Would recommend replacing the 2nd paragraph with: 

"The Seattle Community Network (SCN) is a wireless Internet access network using 4G LTE and WiFi technologies, providing public access from partner locations such as libraries, schools, businesses, and community centers. The Internet connection at these sites is shared wirelessly to nearby devices using the 4G LTE (cell-phone) data standard, which can be used by certain phones and hotspots (also known as Customer Premises Equipment or CPE). Individual users can connect to this signal using SCN-provided (or other compatible) devices to create a local WiFi network in their home. Some of our installed sites utilize a portion of the University of Washington's internet bandwidth, sharing it out to further neighborhoods via point-to-point wireless links. Some sites use other upstream internet service providers (ISPs) such as Lumen."

Reply:
Change completed in e36e9c2.